test/init.spec.js

```diff
@@ -139,6 +140,27 @@ describe("init() method with domain option", () => {
   });
 });
 
+describe("init() method with baseUrl option", () => {
+  it("should add baseUrl to store when provided", async () => {
+    const baseUrl = "https://custom.example.com/api/v1";
+
+    Userfront.init(tenantId, { baseUrl });
+
+    expect(store.baseUrl).toEqual(baseUrl);
+  });
+
+  it("should default baseUrl to 'https://api.userfront.com/v0' if empty or not provided", async () => {
+    // Modify baseUrl to be sure it's changed again in `init`
+    store.baseUrl = "https://example.com";
```

Comment:
Were you able to actually run the library? It seems like this will cause an error, as there is no trailing slash here and no beginning slash everywhere it is used.

Reply:
No I didn't run it locally; sounds like something that the tests should cover. Will update to support URL without trailing slash with an integration tests for cases for both.